.github/ISSUE_TEMPLATE/introduce-sgid-dataset.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Introduce data to the SGID
+about: Creation Checklist
+title: Add <dataset name> from <place>
+labels: 'introduction'
+assignees: ''
+
+---
+# Summary
+
+A short summary of the situation...
+
+<!-- 
+1. The three teams will triage the request, striking through any line that does not apply.
+1. Assign each step to someone based on list of assignments
+1. All checkboxes should either be striked out or checked before closing the issue.
+-->
+
+# Triage
+
+- [ ] Data Team Triage
+- [ ] Dev Team Triage (assigned to)
+- [ ] Cadastre Team Triage
+
+# Data is live in the following areas
+
+- [ ] ArcGIS Online (assigned to)
+- [ ] Internal SGID (assigned to)
+- [ ] SGID10 (assigned to)
+- [ ] Open SGID (assigned to)
+- [ ] Open data (assigned to)
```

Comment:
These are automatic also. Maybe you want someone to just verify that they actually happened?

Reply:
Yes that was the idea and maybe that is part of the confusion.

There are things we have to do when we add data and then there are places the data needs to end up.

I can see splitting this into a I need to do this section and our tools did their job section.

Reply:
I like the idea of splitting these up into action items and stuff to verify.